plans/help.fmf

```diff
@@ -0,0 +1,11 @@
+summary: Run with --help
+prepare:
+  - how: ansible
+    playbook: playbooks/stagingyum.yml
+  - how: install
+    package:
+      # https://github.com/theforeman/foreman-packaging/pull/10670
+      - hostname
+      - foreman-maintain
+execute:
+  - script: foreman-maintain --help
```

Comment:
Do you know why `--help` triggers hostname?

Reply:
No. It's either this:
https://github.com/theforeman/foreman_maintain/blob/cfbb8c444539ee6439619f71c5445ae2a6ddb191/lib/foreman_maintain/concerns/system_helpers.rb#L94-L96

But more likely this:
https://github.com/theforeman/foreman_maintain/blob/cfbb8c444539ee6439619f71c5445ae2a6ddb191/lib/foreman_maintain/config.rb#L24

That latter should really use block form:
```ruby
@foreman_url = @options.fetch(:foreman_url) { `hostname -f`.chomp }
```

That way it doesn't execute `hostname -f` if a value is found.

Reply:
You could, in theory, use this PR to test both this PR and the theory? By including these pieces of code selectively commented out and removing the hostname requires. Kind of a fun experiment.

Reply:
https://github.com/theforeman/foreman_maintain/pull/819. And there's already plenty of other things I'm trying to test. First I just want it to work.